lib/resourcemerge/policyv1.go

```diff
@@ -1,13 +1,13 @@
 package resourcemerge
 
 import (
-	"k8s.io/api/policy/v1beta1"
+	"k8s.io/api/policy/v1"
```

Comment:
optional nit to maintain  `policyv1` naming standard

Reply:
Done.